lib/contentful_model/chainable_queries.rb

```diff
@@ -39,6 +38,14 @@ def locale(locale_code)
         self
       end
 
+      def paginate(page = 1, per_page = 100)
+        page = 1 if page.nil?
+        per_page = 25 if per_page.nil?
+        skip_records_count = (page - 1) * per_page
+        @query << { 'limit' => per_page, 'skip' => skip_records_count }
```

Comment:
Do you think you should add the `order` parameter, to "_keep the ordering predictable_"?

https://www.contentful.com/developers/docs/references/content-delivery-api/#/introduction/collection-resources-and-pagination

Reply:
Yes, will do